install/project/includes/configuration/configuration.inc.sample.php

```diff
@@ -165,9 +166,11 @@
 			define ('__APP_CSS_ASSETS__', __PROJECT_ASSETS__ . '/css');
 			define ('__APP_IMAGE_ASSETS__', __PROJECT_ASSETS__ . '/images');
 			define ('__APP_PHP_ASSETS__', __PROJECT_ASSETS__ . '/php');
+			define ('__APP_IMAGE_CACHE__', __APP_IMAGE_ASSETS__ . '/cache');
+			define ('__APP_UPLOAD__', __DOCROOT__ . __PROJECT_ASSETS__ . '/upload');
```

Comment:
Hmmm, @spekary, you're right, I forgot that security is very important. I'm very sorry. The principle is that it is not good that the folders are a several of places.

Folders to organize and should be separated from this principle, for the function of which is intended for one or the other. It is important only to, at least in one place.

PS. You should definitely take advantage of these folders configuration.inc.php comment.